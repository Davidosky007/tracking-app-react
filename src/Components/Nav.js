const Nav = () => (
  <div className="background-blue nav fixed">
    <h1 className="color-white title bold large">Healthify</h1>
  </div>
);

export default Nav;
