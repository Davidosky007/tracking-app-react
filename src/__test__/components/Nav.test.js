import renderer from 'react-test-renderer';
import Nav from '../../Components/Nav';

describe('Nav', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Nav />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
