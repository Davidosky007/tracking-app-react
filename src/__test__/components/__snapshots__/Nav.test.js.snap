// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<div
  className="background-blue nav fixed"
>
  <h1
    className="color-white title bold large"
  >
    TrackLife
  </h1>
</div>
`;
